tests/main/views/create_a_brief/test_publish.py

```diff
@@ -319,6 +319,7 @@ def test_preview_source_page_shows_disabled_header_breadcrumbs_and_footer_links(
     def test_preview_source_page_will_open_user_generated_links_in_a_new_tab(self):
         brief_json = self._setup_brief()
         brief_json['briefs']['summary'] = "A link to the full summary: https://www.example.com"
+
```

Comment:
Looks like an accidental edit? Can we revert?

Reply:
Done